teps/0112-replace-volumes-with-workspaces.md

```diff
@@ -140,16 +144,16 @@ Some use cases require creating Secrets or ConfigMaps during the execution of th
 subsequent `TaskRuns`. This is not currently possible with Workspaces, as Tekton validates that all Secrets and ConfigMaps used in Workspace
 bindings exist at the time when a `PipelineRun` or `TaskRun` is created.
 
-## Design Considerations
-
-### Task volumes as "implementation details"
+#### Task volumes as "implementation details"
```

Comment:
This could definitely be taken care of with some of the options proposed in https://github.com/tektoncd/community/blob/main/teps/0082-workspace-hinting.md